src/templates/learn-doc.js

```diff
@@ -35,30 +35,43 @@ const ContentRow = ({ children }) => (
 );
 
 const getChildren = (path, navLinks) => {
-  return navLinks.filter(
-    (node) =>
-      node.fields.path.includes(path) &&
-      node.fields.path.split('/').length === path.split('/').length + 1,
-  );
+  return navLinks
+    .filter(
+      (node) =>
+        node.fields.path.includes(path) &&
+        node.fields.path.split('/').length === path.split('/').length + 1,
+    )
+    .sort((a, b) => a.fields.path.localeCompare(b.fields.path));
 };
 
-const Tiles = ({ mdx, navLinks }) => {
+const Tiles = ({ mode, mdx, navLinks }) => {
+  const modes = {
+    None: 'none',
+    Simple: 'simple',
+    Full: 'full',
+  };
+  if (mode === modes.None) return null;
+
   const { path } = mdx.fields;
-  const depth = path.split('/').length;
-  if (depth === 4) {
+
+  if (!mode) {
+    const depth = path.split('/').length;
```

Comment:
`depth` is actually available from graphql now as a field, so `node.fields.depth` should work here

Reply:
does if I add it to the query... Did both (page and nav) to keep them comparable.